sympy/stats/rv.py

```diff
@@ -328,6 +328,15 @@ def free_symbols(self):
             return free_syms
         return {self}
 
+    @property
+    def pspace(self):
+        from sympy.stats.stochastic_process import StochasticPSpace
+        temp_pspace = self.args[1]
+        if len(temp_pspace.args) == 0:
+            # Allow single arg
+            return PSpace()
+        return StochasticPSpace(temp_pspace.args[0], temp_pspace.args[1], temp_pspace.args[2])
```

Comment:
```suggestion
        return temp_pspace
```
Actually, we can replace all these lines by simply temp_pspace now, no need to handle the single arg case separately.